### PR TITLE
feat(shop): added login no servces

### PR DESCRIPTION
**Pull request by m-spilman**
trying to break login into two prs

### CODE REVIEW

File: src/store/login/action.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  return (dispatch) => {
+    dispatch({ type: actionTypes.LOGIN_REQUEST, user: { username } });
+
+    userService.login(username, password).then(
+      ((user) => {
+        dispatch({ type: actionTypes.LOGIN_SUCCESS, user });
+        history.push('/');
+      },
+      (error) => {
+        console.log(error);
```
Comment (BarretteBanner) [inline src/store/login/action.js]: no console.logs

File: src/store/login/action.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        dispatch({ type: actionTypes.LOGIN_FAILURE, error });
+      })
+    );
+  };
+}
```
Comment (BarretteBanner) [inline src/store/login/action.js:24]: add a blank line between functions

File: src/store/store.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import loginReducer from './login/reducer';
+import registerReducer from '.register/reducer';
+import thunkMiddleware from './redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
```
Comment (BarretteBanner) [inline src/store/store.js]: This is super nit picky but a good practice is to have your imports be (dotted lines indicate a blank line) :

from libraries such as redux and react
...
other modules in your application
... 
local imports such as your './login/reducer

**Comment by m-spilman** [comment]
Per our discussion Ahmed, I created userService and added empty functions. - Hope this is what you meant. If not let me know : )

**Comment by m-spilman** [comment]
arg caps. I hope i dont break it when i fix the file name

**Comment by ahaly92** [comment]
nice work! you have my blessing just make sure to incorporate barrette's points

**Review by BarretteBanner — APPROVED** [review]

**Pull request merged**